### DISCUSSION TITLE
command based flight control greyed out with flow deck v2 present

### DISCUSSION THREAD

**Discussion by jjdike** [bitcraze/discussions#1164] [Q&A] [answered]
Hi - I've read a similar discussion to this but am not sure if it is the same problem or solution.  I can use the assist mode to hover with my new crazy flie, crazyradio 2.0, and flow deck v2 configuration, using an xbox controller through a usb connection to the controller.  However the command based flight control is greyed out, which is perhaps the state that box should be in when using a controller.  If I disconnect the xbox controller, this command based flight control is still greyed out.  When is this set of controls supposed to be available?

Thanks.

**Comment by knmcguire** [comment, marked as answer]
Yes indeed, if you connect a controller than the command flight control should be grayed out to avoid the mixing of different setpoint messages.

However, perhaps that upon disconnect of the controller, it should pop back up indeed. Maybe that is a bug.

- On which OS are you running the client on?
 -  If you restart the cfclient without an controller attached, do you see the buttons reappear?

**Reply by jjdike** [reply]
I'm running on windows 11.  If I start cfclient without the controller attached the buttons are active and it flies as expected.  Thanks for the quick reply.

**Reply by knmcguire** [reply]
perfect. Thanks for checking! I'll add this as a feature though for the client: https://github.com/bitcraze/crazyflie-clients-python/issues/697
